app/context-map/om-workload.js

```diff
@@ -0,0 +1,6 @@
+const SourceWorkload = require('../staging/domain/om-workload')
+const assertObjectType = require('../points/domain/validation/assert-object-type')
+
+module.exports = function (sourceWorkload) {
+  assertObjectType(sourceWorkload, SourceWorkload, 'staging/domain/om-workload')
```

Comment:
Just verifying there's no expected return from this and its just asserting that both object models are the same?